cfme/configure/configuration/server_settings.py

```diff
@@ -0,0 +1,416 @@
+        return view.smtp_server.read()
+
+# ============================= Web Services Form =================================
+
+    def update_web_services(self, updates, reset=False):
+        """ Navigate to a Server Tab. Updates web services
+        Args:
+             updates: dict, widgets will be updated regarding updates.
+             reset: By default(False) changes will not be reset, if True changes will be reset
+        """
+        view = navigate_to(self, 'Details')
+        view.web_services.fill(updates)
+        self._save_action(view, updates, reset)
+
+    @property
+    def web_services_form(self):
+        view = navigate_to(self, 'Details')
+        return view.web_services.read()
+
+# ============================= Logging Form =================================
+
+    def update_logging_form(self, updates, reset=False):
+        """ Navigate to a Server Tab. Updates logging form
+        Args:
+             updates: dict, widgets will be updated regarding updates.
+             reset: By default(False) changes will not be reset, if True changes will be reset
+        """
+        view = navigate_to(self, 'Details')
+        view.web_services.fill(updates)
+        self._save_action(view, updates, reset)
+
+    @property
+    def logging_form_form(self):
+        view = navigate_to(self, 'Details')
+        return view.logging_form.read()
+
+# ============================= Custom Support Url Form =================================
+
+    def update_custom_support_url(self, updates, reset=False):
+        """ Navigate to a Server Tab. Updates custom support url
+        Args:
+             updates: dict, widgets will be updated regarding updates.
+             reset: By default(False) changes will not be reset, if True changes will be reset
+        """
+        view = navigate_to(self, 'Details')
+        view.custom_support_url.fill(updates)
+        self._save_action(view, updates, reset)
+
+    @property
+    def custom_support_url_form(self):
+        view = navigate_to(self, 'Details')
+        return view.custom_support_url.read()
+
+    def _save_action(self, view, updates, reset):
+        """ Take care of actions to do after updates """
+        if reset:
+            try:
+                view.reset_button.click()
+                flash_message = 'All changes have been reset'
+            except Exception:
+                logger.warning('No values was changed')
+        else:
+            view.save_button.click()
+            self.appliance.server_details_changed()
+            self._update_att(updates)
+            flash_message = (
+                'Configuration settings saved for {} Server "{} [{}]" in Zone "{}"'.format(
+                    self.appliance.product_name,
+                    self.appliance.server_name(),
+                    self.appliance.server_id(),
+                    self.appliance.server.zone.name))
+        view.flash.assert_message(flash_message)
+
+    def _update_att(self, updates):
```

Comment:
**REQURIED**
Why is this method required? `ServerInformation` class is already `Updateable`, so you don't need that stuff below.